tests/Fixtures/Entity/UuidBinaryEntity.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity;
+
+
+class UuidBinaryEntity
+{
+    private $uuid;
+
+    public function __construct($uuid)
```

Comment:
Can you add typehints and return types here?

Reply:
done